_data/education.yml

```diff
@@ -51,6 +51,13 @@ websites:
       twitter: duolingo
       tfa: No
 
+    - name: EC-Council
+      url: https://www.eccouncil.org
+      img: eccouncil.png
+      tfa: Yes
+      sms: Yes
+      doc: https://aspen.eccouncil.org/aspen/Help/2FactorAuthentication-User_Manual.pdf
```

Comment:
Is the documentation available outside of a PDF file? I have been under the assumption that links to PDF files are to be avoided if at all possible.